bootstrap/011_warehouse_schedules.sql

```diff
@@ -10,23 +10,36 @@ CREATE OR REPLACE PROCEDURE INTERNAL.UPDATE_WAREHOUSE_SCHEDULES(last_run timesta
 DECLARE
     task_outcome variant default (select object_construct());
 BEGIN
+    -- Get the configured timezone or default to 'America/Los_Angeles'
+    let tz text;
+    call internal.get_config('default_timezone') into :tz;
+    if (tz is null) then
+        tz := 'America/Los_Angeles';
+    end if;
+    task_outcome := (select object_insert(:task_outcome, 'opscenter timezone', :tz));
```

Comment:
I am not sure why we have two lines here. as in they are basically the same info right?

Reply:
Ah, one is recording what the configured timezone for OpsCenter is and the other is recording what the default timezone is for the user's Snowflake account. I thought knowing this might be helpful in debugging issues in the future.